conda/recipes/libcuml/conda_build_config.yaml

```diff
@@ -0,0 +1,17 @@
+cmake_version:
+  - ">=3.20.1,<3.23"
+
+nccl_version:
+  - ">=2.9.9"
+
+treelite_version:
```

Comment:
Quick question for the future, the versions of this file would be also potentially used by the `cuml` conda recipe, no? Particularly `treelite` for example, is there a way to put this file in `conda/recipes` so that it could be shared?

Reply:
Yes, absolutely. We can use the `--variant-config-files`/`-m` flag in the `conda build` command to point to a particular file that has all of the version specifications. This is what's done in the `integration` repo's `versions.yaml` file. Here's the official `conda` docs on that file for reference:

- https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#creating-conda-build-variant-config-files

We can implement this as a future enhancement since we have some additional CI changes coming soon.